Implemented Get Messsage feature in wchat

Rits1272 opened the pull request:
## Issue that this pull request solves

Closes: #77 

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

![Screenshot (61)](https://user-images.githubusercontent.com/44135362/78914409-f95d7480-7aa7-11ea-837c-8c21aac5dab2.png)

## Other information

A user can now receive messages in `wchat` . 
The script will be continuously listening to messages from the target. If a message is sent by the target it will be displayed in the terminal. If a user wants to send a message he simply needs to press `space bar`. After that, the user can send his message.

Initially, @xandao6 suggested using threading. But since `asyncio` is itself always single-threaded it was no feasible to add a thread. So I came out with this solution.

Rits1272 commented:
Please review @xandao6 @rpotter12 @priyansh19. Sorry for sending PR late. It actually took me a while to come out with the solution.

rpotter12 commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- wplay/terminal_chat.py  3
         

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824&bid=17466134)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -31,22 +33,56 @@
+                    message.remove('#_FILE')
+                    await io.send_file(page)
+                await io.send_message(page, message)
+        except:
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Try, Except, Pass detected.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -31,22 +33,56 @@
+    if tg.lower() in last_message_sender.lower() and lastOutgoingMessage!=lastMessage:
+        print(Fore.GREEN + f"{tg}-", end="")
+        print(lastMessage, end="")
+        print(Style.RESET_ALL) 
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -31,22 +33,56 @@
+                        await target_search.search_and_select_target(page, target)
+                    else:
+                        await target_select.manual_select_target(page)
+                    message = io.ask_user_for_message_breakline_mode()       
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

Rits1272 commented:
Hello @rpotter12 I have mentioned in the screenshot as well as in the description if you want to send message you first need to hit `spacebar`. Kindly check.

xandao6 commented:
add keyboard library to setup.py also.

xandao6 commented:
![image](https://user-images.githubusercontent.com/22106326/78949661-94ebe680-7aa2-11ea-9dc1-f60d0d11210c.png)

xandao6 commented:
I got a lot of bugs using 'q' to exit,

rpotter12 commented:
> Hello @rpotter12 I have mentioned in the screenshot as well as in the description if you want to send message you first need to hit `spacebar`. Kindly check :)

@Ritz1272 I am still not able to send message from your implementation. Please check your implementation again and test it again.

rpotter12 commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- wplay/terminal_chat.py  2
         

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824&bid=17466134)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -27,26 +29,58 @@ async def chat(target):
+    if tg.lower() in last_message_sender.lower() and lastOutgoingMessage!=lastMessage:
+        print(Fore.GREEN + f"{tg}-", end="")
+        print(lastMessage, end="")
+        print(Style.RESET_ALL) 
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -27,26 +29,58 @@ async def chat(target):
+                        await target_search.search_and_select_target(page, target)
+                    else:
+                        await target_select.manual_select_target(page)
+                    message = io.ask_user_for_message_breakline_mode()       
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -27,26 +29,58 @@ async def chat(target):
+                    message.remove('#_FILE')
+                    await io.send_file(page)
+                await io.send_message(page, message)
+        except:
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Try, Except, Pass detected.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

Rits1272 commented:
- Manual target select does not give any errors
- Added `keyboard` library in the `setup.py` file
-  Removed the option to exit the terminal chat by pressing `q`

Please review @rpotter12 @xandao6

rpotter12 commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- wplay/terminal_chat.py  2
         

Clones removed
==============
+ wplay/terminal_chat.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824&bid=17466134)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -25,31 +27,60 @@ async def chat(target):
+    if tg.lower() in last_message_sender.lower() and lastOutgoingMessage!=lastMessage:
+        print(Fore.GREEN + f"{tg}-", end="")
+        print(lastMessage, end="")
+        print(Style.RESET_ALL) 
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -25,31 +27,60 @@ async def chat(target):
+                    message.remove('#_FILE')
+                    await io.send_file(page)
+                await io.send_message(page, message)
+        except:
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 replied:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Try, Except, Pass detected.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented on wplay/terminal_chat.py:
```diff
@@ -25,31 +27,60 @@ async def chat(target):
+                        await target_search.search_and_select_target(page, target)
+                    else:
+                        await target_select.manual_select_target(page)
+                    message = io.ask_user_for_message_breakline_mode()       
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5335824)

rpotter12 commented:
![Screenshot from 2020-04-17 17-15-34](https://user-images.githubusercontent.com/34352567/79565932-47d5c900-80cf-11ea-936b-53e09c474942.png)

My screenshot doesnot match with your screenshot. `write your message (Enter key for breakline) ('.' alone to send message)` was not found when I ran the software. I think this was your old screenshot.
I am still not able to send message
